public/blocking.css

```diff
@@ -0,0 +1,103 @@
E3MTg4LDI4LjI3MzQzOCBjIC0xLjI2Nzg5NDg1LDIuMTM5Mjk2IC0xLjI2NzIzMjE1LDQuODExNTk3IDAsNi45NTExNzEgMy4zMTI2ZS00LDUuNmUtNCAtMy4zMTQ0ZS00LDAuMDAxNCAwLDAuMDAyIEwgMTMuNDQ1MzEyLDU2LjUxOTUzMSBDIDE0LjY4NzkxMSw1OC42ODcxODggMTcuMDA5MDM1LDYwIDE5LjQ5MjE4OCw2MCBoIDI0Ljk5ODA0NiBjIDIuNDgzMTUzLDAgNC43OTAwMzEsLTEuMzA1MDA1IDYuMDUyNzM1LC0zLjQzNTU0NyBsIDAuMDAyLC0wLjAwMzkgMTIuNTA5NzY2LC0yMS4zMjQyMTkgYyAxLjI1NjQ3OSwtMi4xNDYyMDkgMS4yNjI3MDUsLTQuODE0ODMyIDAsLTYuOTUzMTI1IGwgLTAuMDAyLC0wLjAwMzkgLTEyLjQ4ODI4MSwtMjEuMzM3ODkwOCAtMC4wMDM5LC0wLjAwNTg2IEMgNDkuMjk3ODQzLDQuODA1MDA4NyA0Ni45OTA5NjcsMy41IDQ0LjUwNzgxMiwzLjUgWiBtIDAsNCBoIDI1LjAxNTYyNCBjIDEuMDc5ODI2LDAgMi4wNzM3OSwwLjU2OTU4MzMgMi42MTEzMjksMS40NzY1NjI1IGwgMTIuNDg2MzI4LDIxLjMzMjAzMTUgMC4wMDM5LDAuMDAzOSBjIDAuNTMyMDE2LDAuODk3NjYxIDAuNTM2MDgsMS45NzY2NTEgLTAuMDA1OSwyLjkwMjM0NCBsIC0xMi41MDE5NTQsMjEuMzA4NTk0IC0wLjAwMzksMC4wMDIgQyA0Ni41NTk2MTYsNTUuNDMwMjM2IDQ1LjU2ODYwNCw1NiA0NC40OTAyMzQsNTYgSCAxOS40OTIxODggYyAtMS4wNzk4MjYsMCAtMi4wNTc5ODYsLTAuNTYxMzkgLTIuNTc4MTI2LC0xLjQ2ODc1IGwgLTAuMDAzOSwtMC4wMDk4IC0xMi41MTU2MjQ4LC0yMS4zMzIwMzEgLTAuMDAxOTUsLTAuMDAyIGMgLTAuNTMyMDE1NywtMC44OTc2NjEgLTAuNTMyMDE1NywtMS45NzczMzkgMCwtMi44NzUgbCAwLjAwMTk1LC0wLjAwMzkgTCAxNi44ODA4NTksOC45NzY1NjI1IEMgMTcuNDE4Mzk5LDguMDY5NTgzIDE4LjQxMjM2Myw3LjUgMTkuNDkyMTg4LDcuNSBaIi8+PC9jbGlwUGF0aD48L2RlZnM+PHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBkPSJtIDE5LjQ5MjE4OCwzLjUgYyAtMi40ODMxNTQsMCAtNC43OTAwMywxLjMwNTAwOSAtNi4wNTI3MzUsMy40MzU1NDY5IGwgLTAuMDAyLDAuMDA1ODYgTCAwLjk1MTE3MTg4LDI4LjI3MzQzOCBjIC0xLjI2Nzg5NDg1LDIuMTM5Mjk2IC0xLjI2NzIzMjE1LDQuODExNTk3IDAsNi45NTExNzEgMy4zMTI2ZS00LDUuNmUtNCAtMy4zMTQ0ZS00LDAuMDAxNCAwLDAuMDAyIEwgMTMuNDQ1MzEyLDU2LjUxOTUzMSBDIDE0LjY4NzkxMSw1OC42ODcxODggMTcuMDA5MDM1LDYwIDE5LjQ5MjE4OCw2MCBoIDI0Ljk5ODA0NiBjIDIuNDgzMTUzLDAgNC43OTAwMzEsLTEuMzA1MDA1IDYuMDUyNzM1LC0zLjQzNTU0NyBsIDAuMDAyLC0wLjAwMzkgMTIuNTA5NzY2LC0yMS4zMjQyMTkgYyAxLjI1NjQ3OSwtMi4xNDYyMDkgMS4yNjI3MDUsLTQuODE0ODMyIDAsLTYuOTUzMTI1IGwgLTAuMDAyLC0wLjAwMzkgLTEyLjQ4ODI4MSwtMjEuMzM3ODkwOCAtMC4wMDM5LC0wLjAwNTg2IEMgNDkuMjk3ODQzLDQuODA1MDA4NyA0Ni45OTA5NjcsMy41IDQ0LjUwNzgxMiwzLjUgWiBtIDAsNCBoIDI1LjAxNTYyNCBjIDEuMDc5ODI2LDAgMi4wNzM3OSwwLjU2OTU4MzMgMi42MTEzMjksMS40NzY1NjI1IGwgMTIuNDg2MzI4LDIxLjMzMjAzMTUgMC4wMDM5LDAuMDAzOSBjIDAuNTMyMDE2LDAuODk3NjYxIDAuNTM2MDgsMS45NzY2NTEgLTAuMDA1OSwyLjkwMjM0NCBsIC0xMi41MDE5NTQsMjEuMzA4NTk0IC0wLjAwMzksMC4wMDIgQyA0Ni41NTk2MTYsNTUuNDMwMjM2IDQ1LjU2ODYwNCw1NiA0NC40OTAyMzQsNTYgSCAxOS40OTIxODggYyAtMS4wNzk4MjYsMCAtMi4wNTc5ODYsLTAuNTYxMzkgLTIuNTc4MTI2LC0xLjQ2ODc1IGwgLTAuMDAzOSwtMC4wMDk4IC0xMi41MTU2MjQ4LC0yMS4zMzIwMzEgLTAuMDAxOTUsLTAuMDAyIGMgLTAuNTMyMDE1NywtMC44OTc2NjEgLTAuNTMyMDE1NywtMS45NzczMzkgMCwtMi44NzUgbCAwLjAwMTk1LC0wLjAwMzkgTCAxNi44ODA4NTksOC45NzY1NjI1IEMgMTcuNDE4Mzk5LDguMDY5NTgzIDE4LjQxMjM2Myw3LjUgMTkuNDkyMTg4LDcuNSBaIiBmaWxsPSIjZmZmIiBvcGFjaXR5PSIuMiIvPjxnIGNsaXAtcGF0aD0idXJsKCNoZXhDbGlwKSI+PGNpcmNsZSBpZD0iY2lyY2xlIiBjeD0iMzIiIGN5PSIzMiIgcj0iMTYiIGZpbGw9Im5vbmUiIHN0cm9rZS13aWR0aD0iMzIiIHN0cm9rZT0iIzFERTlCNiIgc3Ryb2tlLWRhc2hhcnJheT0iMTYuNjY2IDg0LjY2NiIvPjwvZz48L3N2Zz4=') !important;
```

Comment:
Is `!important` at the end here really necessary?

Reply:
Otherwise it uses the non-working version of @nimiq/style, so yes :( But I'll check again now...

Reply:
Seems to work now, most probably because something you changed in the LoadingSpinner component. So I removed the !important :)